storage/innobase/btr/btr0bulk.cc

```diff
@@ -58,19 +58,11 @@ PageBulk::init()
 	mtr = static_cast<mtr_t*>(
 		mem_heap_alloc(m_heap, sizeof(mtr_t)));
 	mtr_start(mtr);
-<<<<<<< HEAD
```

Comment:
I believe https://github.com/percona/percona-server/commit/7ebaef8e255afe3ef88ee319871e2e51b322e2c7#diff-1b7266575f084a759d5bee343efe91d0R7022 should also be reverted

Reply:
It's also added by upstream at https://github.com/mysql/mysql-server/commit/f9fb96c20f#diff-1b7266575f084a759d5bee343efe91d0R6793

Reply:
Sorry, was looking a the wrong branch